components/ws-daemon/pkg/controller/workspace_operations.go

```diff
@@ -345,9 +344,9 @@ func ensureCleanSlate(location string) error {
 	return nil
 }
 
-func (wso *DefaultWorkspaceOperations) uploadWorkspaceLogs(ctx context.Context, opts BackupOptions) (err error) {
+func (wso *DefaultWorkspaceOperations) uploadWorkspaceLogs(ctx context.Context, opts BackupOptions, location string) (err error) {
```

Comment:
Any unit tests we need to add/update as part of this to ensure the regression doesn't happen again in the future?

Reply:
I've updated an existing integration test to check for uploaded logs. There's no unit tests nor is it easy to add one now that would catch this regression